src/getBullsAndCows.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+function getBullsAndCows(correctNumber, userNumber) {
+  let bulls = 0;
+  let cows = 0;
+
+  for (let i = 0; i < correctNumber.length; i++) {
+    const hasCorrectDigit = correctNumber.includes(userNumber[i]);
+
+    if (hasCorrectDigit && correctNumber[i] === userNumber[i]) {
+      bulls++;
+    }
+
+    if (hasCorrectDigit && correctNumber[i] !== userNumber[i]) {
+      cows++;
+    }
+  }
+
+  return [bulls, cows];
```

Comment:
Better to return object `{bulls, cows}`
So the order doesn't matter when you do destructuring